webpack.config.ts

```diff
@@ -120,7 +129,7 @@ module.exports = (env: string, argv: { [key: string]: string }): Configuration =
             },
          }),
       );
-   } else {
+   } else if (isProd()) {
```

Comment:
I restricted your GeneratePackageJsonPlugin only in Prod. Are you ok with that? I think it is not necessary to have while running locally.

Reply:
yeah sure no prob